test/acceptance/README.md

```diff
@@ -0,0 +1,120 @@
+# FFC SFI Apply Web Acceptance Tests
+
+> Future Farming and Countryside Programme - SFI Apply Web Acceptance Tests.
+
+This folder contains the acceptance tests for the FFC SFI Apply Web 'customer
+facing' `ffc-sfi-apply-web` front end microservice.
+
+The tests use [Cucumber](https://cucumber.io/) and
+[webdriver.io](https://webdriver.io/). It is based on the
+[webdriverio/cucumber-boilerplate](https://github.com/webdriverio/cucumber-boilerplate)
+project.
+The repo has been containerised so it is easier to run in development and
+CI environments.
+
+BrowserStack is used to run the tests to reduce complexity and provide an
+improved experience that would otherwise be possible running tests locally.
+
+## Setup
+
+Need to have environment variables made available to the container. Created `.env` file in acceptance test directory.
+
+Env vars:
+
+TODO: add table
```

Comment:
Is this not already done below?

Reply:
It is. Let me get that updated.

Reply:
Good spot. I'd meant to remove that section as it was to remind me to add it later in the file.
It has now been removed.